lib/src/scss-shared/_nx-tag-variables.scss

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@use '../scss-shared/nx-dark-mode-helpers';
+
+.nx-selectable-color--light-blue, // deprecated
+.nx-selectable-color--sky {
+  --nx-tag-selected-hover-border: var(--nx-color-interactive-border-hover);
```

Comment:
Should these all be `--nx-color-tag-...`?

Reply:
> Should these all be `--nx-color-tag-...`?

Yup. 3899f851770c116ed1c1c254ce3e000215ebb3aa